flanker/addresslib/address.py

```diff
@@ -558,7 +558,7 @@ def to_unicode(self):
         Converts to unicode.
         """
         if self.display_name:
-            return u'{} <{}@{}>'.format(self.display_name, self.local_part, self.domain)
+            return u'{} <{}@{}>'.format(smart_quote(self.display_name), self.local_part, self.domain)
```

Comment:
Cool I am glad I was wrong :). Can you add at least one test that makes that obvious? It would be nice to actually see what quoted display name looks like.